frontend/src/pages/journal/styles.module.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+  box-sizing: border-box;
+  margin-left: 350px;
```

Comment:
can we add this as padding to sidebar and navigation menu instead, to define it in one place?

Reply:
in this way it will looke like this:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/191ea27f-4223-49fc-92c8-8e17677feb21)